package.json

```diff
@@ -49,8 +52,10 @@
     "@telokys/eslint-config-react-typescript": "^2.0.5",
     "@types/lz-string": "^1.3.34",
     "eslint": "^8.28.0",
+    "husky": "^8.0.2",
```

Comment:
I'm not a fan of Husky, I prefer the checks to be on the server side instead.

Reply:
Really, so the push would fail, but you already would have a commit with issues.
And this way auto-fixable issues would already be resolved on commit.

Reply:
I know you could also configure your IDE to do it on-save and if that is the desired change, I'll remove it, too.

Reply:
And gone...